lib/core.js

```diff
@@ -334,7 +338,7 @@ declare class String {
     static fromCharCode(...codes: Array<number>): string;
     static fromCodePoint(...codes: Array<number>): string;
     static raw(templateString: string): string;
-    static raw(callSite: $Shape<{raw: string}>, ...substitutions: any[]): string;
+    static raw(callSite: TemplateStringsArray, ...substitutions: any[]): string;
```

Comment:
`static raw(templateString: string): string;` is invalid (`String.raw('str')` throws an error).

Probably `raw` declaration should be like this:

```js
  static raw(callSite: { +raw: $ReadOnlyArray<string>; ... }, ...substitutions: any[]): string;
```

By this It also accepts a `TemplateStringsArray` instance and objects like `{ raw: ['ab', 'cd'] }`.

Test cases:

```js
String.raw`str` // ok
String.raw`a${2}b` // ok
String.raw({ raw: ['ab', 'cd'] }, 1) // ok
String.raw('str') // error
String.raw({ k: 'v' }) // error
```